src/components/calculator/SelectControl.tsx

```diff
@@ -1,16 +1,31 @@
+import num2fraction from 'num2fraction'
 import React from 'react'
 import { Badge, OverlayTrigger } from 'react-bootstrap'
 
 import { CalculatorData } from 'data/calculate'
 import { FormValue } from 'data/data'
 
+function showRiskMultiplier(multiplier: number): string {
+  // console.log(multiplier)
+  // console.log(multiplier < 0)
+  // debugger
+  if (multiplier === 1) {
+    return 'default risk'
```

Comment:
I think I prefer [1x risk]. "Default" and "1x" are both showing that this is an implementation detail, so I'd go with the more consistent format.

Likewise, maybe do `num2fraction x risk`

_Additionally_ maybe lets do "transmission" or "chance of transmission" instead of "risk" as someone pointed out that we have used "risk" to mean way too many things.

Reply:
Regarding 1x risk: Definitely is an implementation detail. I don't know that "1x risk" means anything to a lot of people. It feels confusing to me if I put myself in the seat of a user. That said, I think "baseline risk" might be better than "default risk." Whatcha think?

> Likewise, maybe do num2fraction x risk

Do you mean doing "1/20 x risk" instead of "1/20th the risk"

> "chance of transmission" 

Hmmm. Makes the select boxes really long, which is challenging on mobile. Maybe we could switch to this once we implement better boxes (see #298).

Reply:
I think it's more cognitive load to see descriptors with different formats. Parallel structure!

I'm ok with "risk" but want to challenge us to find better but concise language.

Reply:
@beshaya Can you clarify exactly what you'd have them say, then?

Reply:
Agreed about finding better language!

Reply:
Also, I was following @mattbell's suggested format from the original issue when using "default risk" vs "5x risk". I'm curious if he wants to weigh in on this.